### PR TITLE
First commit for getting the vuln details.

### PR DESCRIPTION
**Pull request by rick-kenna**
## Problem
Needed a vuln details code exampl

## Solution
Provide a vuln details code examlel

### CODE REVIEW

File: python/vuln_details/blog_get_vuln_details.py [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    print("There are 2 formats:")
+    print(f"    {prog_name}")
+    print(f"    {prog_name} <export_search_id>")
+    print("Where <export_search_id> is search ID from an vuln export.")
+    print("If <export_search_id> is not present, a new export is created and retrieved.")
+    print("")
+    print("There are 3 output files:")
+    print("    vulns_<export_id>.jsonl")
+    print("    vulns_<export_id>.json")
+    print("    vulns_<export_id>.html")
```
Comment (hingandh) [inline python/vuln_details/blog_get_vuln_details.py:37]: xml and not html?

File: python/vuln_details/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Vulnerabilities' Details Code Examples
+
+## Get Vulneraibilities' Details with Field Selection
+
+* blog_get_vuln_details.py - Obtains the vulnerability details along with some other fields and create three files, one in JSONL, one in formated JSON, and one in HTML.
```
Comment (hingandh) [inline python/vuln_details/README.md:5]: and one in XML?

Comment (rick-kenna) [reply python/vuln_details/README.md:5]: No XML.  The code exports JSONL, and then pretty formats the JSON and HTML.

**Review by hingandh — APPROVED** [review]

**Review by ababhis2 — APPROVED** [review]
Looks good 🚀

File: python/vuln_details/blog_get_vuln_details.py [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+    # Initialize the HTML Sanitizer and open the output files.
+    sanitizer = Sanitizer()  # default configuration
+    json_f = open(json_vuln_file_name, 'w')
+    html_f = open(html_vuln_file_name, 'w')
+
+    # Read the JSONL input file and write out pretty JSON and HTML files.
+    num_lines = 0
+    with open(jsonl_vuln_file_name, 'r') as jsonl_f:
+        for line_num, vuln_line in enumerate(jsonl_f):
```
Comment (jaredkalmus) [inline python/vuln_details/blog_get_vuln_details.py:318]: Receiving an error here. Likely due to crappy data in the details field. Additional sanitization may be needed.

`(pandas_env) PS C:\Users\jaredk\OneDrive - Cisco\Documents\Scripts\vuln_details_export> python blog_get_vuln_details.py Vulnerabiity Exports with Details
New export search ID: 2909784 with 4607 vulnerabilities.
Sleeping for 5 seconds.  (16 out of 240)
Gunzipping file vulns_2909784.gz to vulns_2909784.jsonl
File vulns_2909784.gz gunzipped to vulns_2909784.jsonl
Counting lines in vulns_2909784.jsonl
File: vulns_2909784.jsonl with 4607 vulns.
Traceback (most recent call last):
  File "C:\Users\jaredk\OneDrive - Cisco\Documents\Scripts\vuln_details_export\blog_get_vuln_details.py", line 318, in <module>
    for line_num, vuln_line in enumerate(jsonl_f):
  File "C:\Users\jaredk\Miniconda3\envs\pandas_env\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 7268: character maps to <undefined> `

File: python/vuln_details/blog_get_vuln_details.py [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+            json_f.write(f"{vuln_formatted}\n")
+            vuln_formatted = htmlize_vuln_with_details(vuln, sanitizer)
+            html_f.write(f"{vuln_formatted}<p>")
+
+            num_lines += 1
+
+    # Close up shop and publish results.
+    json_f.close()
+    html_f.close()
+    print(f"All pau.")
```
Comment (jaredkalmus) [inline python/vuln_details/blog_get_vuln_details.py:331]: Not sure what "pau" means -- is this meant to be "paused"?

**Review by jaredkalmus — CHANGES_REQUESTED** [review]
Please let me know when the decoding error is resolved and I'll retest

File: python/vuln_details/blog_get_vuln_details.py (hunk 3 above)
Comment (ababhis2) [reply python/vuln_details/blog_get_vuln_details.py:318]: this is going to be a problem with details from json export and it still exists in Scanner Vulnerability Details API endpoint as well.

File: python/vuln_details/blog_get_vuln_details.py (hunk 4 above)
Comment (rick-kenna) [reply python/vuln_details/blog_get_vuln_details.py:331]: It Hawai'ian for done.

Comment (jaredkalmus) [reply python/vuln_details/blog_get_vuln_details.py:331]: I like it. Feel free to keep it if you want :)

**Pull request merged**